### ISSUE TITLE
in docker image, rclone zombie processes grow up

### ISSUE THREAD

**Issue by lzecca78** [garethgeorge/backrest#225]
Hi 👋 ,
i am using `garethgeorge/backrest:latest-alpine` on debian 12.5 and I've noticed that the more the process run rclone processes, the more `rclone` zombie processes are left as you can see in the following log, and the number is growing everytime a new rclone schedule is executed. 
I am using a repository that relies on rclone backend because is not handled directly by restic (in the specific case is pcloud)

```
2246847 ?        Sl     5:35 /usr/bin/containerd-shim-runc-v2 -namespace moby -id 880d8af6dbcaf92d8b6749dab0b2d5596b7cc8f4d3905758f3ddf0033e098a16 -address /run/containerd/containerd.sock
2246868 ?        Ssl    0:07  \_ /backrest
2343525 ?        Z      0:00      \_ [rclone] <defunct>
2343553 ?        Z      0:01      \_ [rclone] <defunct>
2343629 ?        Z      0:00      \_ [rclone] <defunct>
2343671 ?        Z      0:00      \_ [rclone] <defunct>
2343704 ?        Z      0:00      \_ [rclone] <defunct>
2535089 ?        Z      0:00      \_ [rclone] <defunct>
2535117 ?        Z      0:01      \_ [rclone] <defunct>
2535204 ?        Z      0:00      \_ [rclone] <defunct>
2535229 ?        Z      0:00      \_ [rclone] <defunct>
2535282 ?        Z      0:00      \_ [rclone] <defunct>
2695685 ?        Z      0:00      \_ [rclone] <defunct>
2695724 ?        Z      0:01      \_ [rclone] <defunct>
2695854 ?        Z      0:00      \_ [rclone] <defunct>
2695879 ?        Z      0:00      \_ [rclone] <defunct>
2695913 ?        Z      0:00      \_ [rclone] <defunct>
2883731 ?        Z      0:00      \_ [rclone] <defunct>
2883760 ?        Z      0:01      \_ [rclone] <defunct>
2883856 ?        Z      0:00      \_ [rclone] <defunct>
2883882 ?        Z      0:00      \_ [rclone] <defunct>
2883910 ?        Z      0:00      \_ [rclone] <defunct>
3042682 ?        Z      0:00      \_ [rclone] <defunct>
3042713 ?        Z      0:01      \_ [rclone] <defunct>
3042801 ?        Z      0:00      \_ [rclone] <defunct>
3042827 ?        Z      0:00      \_ [rclone] <defunct>
3042859 ?        Z      0:00      \_ [rclone] <defunct>
3190562 ?        Z      0:00      \_ [rclone] <defunct>
3190701 ?        Z      0:00      \_ [rclone] <defunct>
3233167 ?        Z      0:00      \_ [rclone] <defunct>
3233199 ?        Z      0:01      \_ [rclone] <defunct>
3233300 ?        Z      0:00      \_ [rclone] <defunct>
3233341 ?        Z      0:00      \_ [rclone] <defunct>
3233369 ?        Z      0:00      \_ [rclone] <defunct>
3391152 ?        Z      0:00      \_ [rclone] <defunct>
3391202 ?        Z      0:01      \_ [rclone] <defunct>
3391270 ?        Z      0:00      \_ [rclone] <defunct>
3391323 ?        Z      0:00      \_ [rclone] <defunct>
3391357 ?        Z      0:00      \_ [rclone] <defunct>
3500288 ?        Z      0:00      \_ [rclone] <defunct>
3500316 ?        Z      0:00      \_ [rclone] <defunct>
3500353 ?        Z      0:00      \_ [rclone] <defunct>
3500476 ?        Z      0:00      \_ [rclone] <defunct>
3500508 ?        Z      0:00      \_ [rclone] <defunct>
3577716 ?        Z      0:00      \_ [rclone] <defunct>
3577744 ?        Z      0:01      \_ [rclone] <defunct>
3577840 ?        Z      0:00      \_ [rclone] <defunct>
3577863 ?        Z      0:00      \_ [rclone] <defunct>
3577891 ?        Z      0:00      \_ [rclone] <defunct>
3735718 ?        Z      0:00      \_ [rclone] <defunct>
3735746 ?        Z      0:01      \_ [rclone] <defunct>
3735806 ?        Z      0:00      \_ [rclone] <defunct>
3735831 ?        Z      0:00      \_ [rclone] <defunct>
3735860 ?        Z      0:00      \_ [rclone] <defunct>
```

**Comment by garethgeorge** [comment]
Thanks for catching and reporting this, I think this is because there isn't any init process in the docker container to reap zombie processes. Yikes

Should be addressed by adding https://github.com/krallin/tini , this is pretty high priority. I'll aim to add that in and put out a patch release tonight.

**Comment by garethgeorge** [comment]
Just released [0.17.2](https://github.com/garethgeorge/backrest/pull/218) as a patch release addressing this.
